src/main/java/ui/MenuControl.java

```diff
@@ -353,6 +353,17 @@ private void updateRepoRemoveList(Menu remove) {
             removeItem.setOnAction(e1 -> onRepoRemove(repoId));
             remove.getItems().add(removeItem);
         }
+
+        // Supposedly, we would like the menu not to close when the disabled MenuItem-s
+        // below are clicked. But this is a JDK bug; we can use CustomMenuItem.setHideOnClick(false)
+        // if we want to. The bug is that it only works for ContextMenu and not Menu (which
+        // we are using).
+        for (String usedRepoId : currentlyUsedRepos) {
+            MenuItem disabledRemoveItem = new MenuItem(usedRepoId + " (in use; non-removable)");
```

Comment:
I should think so? One other option would be to add the `(In Use)` but not disable it, so that when the user clicks on it, it popups up a dialog telling the user why it can't be removed? But that seems even more convoluted. 

Ah, what if, at the very top, you have a disabled menu item that says that repos in use cannot be deleted?